CMake/VerStubUtil.cmake

```diff
@@ -0,0 +1,8 @@
+include("${CMAKE_CURRENT_LIST_DIR}/SMDefs.cmake")
+
+string(TIMESTAMP SM_TIMESTAMP_DATE "%Y%m%d")
+string(TIMESTAMP SM_TIMESTAMP_TIME "%H:%M:%S" UTC)
```

Comment:
Wasn't it already established that this would require cmake 3 or higher? This is a deal breaker.